pkg/controller/daemon/daemon_controller.go

```diff
@@ -261,6 +277,24 @@ func (dsc *DaemonSetsController) Run(workers int, stopCh <-chan struct{}) {
 		go wait.Until(dsc.runWorker, time.Second, stopCh)
 	}
 
+	// Collect garbage
+	go func() {
+		intervalCh := make(chan struct{}, 1)
```

Comment:
defer close ?

Reply:
I'll need to make sure it won't get closed before the other goroutine stops on stopCh close

Reply:
actually I was wrong yesterday - Golang channels don't need to be closed unless a receiver relays on that and are GC'd